code/datums/jobs/job/job.dm

```diff
@@ -1,11 +1,11 @@
 GLOBAL_LIST_INIT(exp_jobsmap, list(
```

Comment:
move this to make_datum_refs since you're referencing a GLOB inside a GLOBAL_LIST_INIT, works now but its a ticking time bomb

Reply:
As said on discord, these requests are getting quite out of scope.